package/android/src/main/java/com/mrousavy/camera/core/outputs/CameraOutputs.kt

```diff
@@ -101,7 +104,11 @@ class CameraOutputs(
     // Preview output: Low resolution repeating images (SurfaceView)
     if (preview != null) {
       Log.i(TAG, "Adding native preview view output.")
-      previewOutput = SurfaceOutput(preview.surface, characteristics.getPreviewSize(), SurfaceOutput.OutputType.PREVIEW)
+      val previewSizeAspectRatio = if (preview.targetSize != null) preview.targetSize.bigger.toDouble() / preview.targetSize.smaller else null
```

Comment:
I'm not sure if we should pass aspect ratios around - it just feels a bit dangerous to have `aspectRatio: Int` being passed, because:

1. We lose orientation (no information if this is `width / height` or `height / width`)
2. We lose context - is it the video's aspect ratio? photo's? preview view's?

Is it possible that we can somehow simplify this, or do you think that's required?

Reply:
We can still change this to have like `getPreviewTargetSizeFromFormatSize`. We don't **need** to have an aspect ratio if we can compute it from the size!